python/ray/serve/_private/constants.py

```diff
@@ -203,10 +203,17 @@ class ServeHandleType(str, Enum):
 
 # Feature flag to enable StreamingResponse support.
 # When turned on, *all* HTTP responses will use Ray streaming object refs.
+# Turning this FF on also enables RAY_SERVE_ENABLE_NEW_ROUTING.
 RAY_SERVE_ENABLE_EXPERIMENTAL_STREAMING = (
     os.environ.get("RAY_SERVE_ENABLE_EXPERIMENTAL_STREAMING", "0") == "1"
 )
 
+# Feature flag to enable power of two choices routing.
+RAY_SERVE_ENABLE_NEW_ROUTING = (
+    os.environ.get("RAY_SERVE_ENABLE_NEW_ROUTING", "0") == "1"
```

Comment:
I chose something vague intentionally because otherwise from my experience users will read too much into the details. This should hopefully never need to be seen/touched by users anyways.